RST_DOC_STYLE.rst

```diff
@@ -0,0 +1,247 @@
+Recommended RST Document Style
+==============================
+
+This document is basically a cheat sheet for reStructuredText to help guide new people in following TurnKeyLinux's documenting style.
+
+Contents
+--------
+
+* `Headers <#Headers>`_
+
+* `Horizontal Rules <#horizontal-rules>`_
+
+* `Code Formatting <#code-formatting>`_
+
+* `Basic Formatting <#basic-formatting>`_
+
+* `Code Blocks <#code-blocks>`_
+
+* `Escaping Markup <#escaping-markup>`_
+
+* `Lists <#Lists>`_
+
+* `Linking <#linking>`_
+
+* `References <#references>`_
+
+Headers
+-------
+
+Typically ``Header 1`` is used for README.rst, any additional documentation pages will start with ``Header 2``, then ``Header 3`` for page sections and ``Header 4`` for subsections. ``Header 5`` is rarely used.
+
+The underlining needs to be atleast as long as the text.
+
+Example of headers::
+
+    ========
+    Header 1
+    ========
+
+    Header 2
+    ========
+
+    Header 3
+    --------
+
+    Header 3 with a longer text
+    ---------------------------
+
+    Header 4
+    ^^^^^^^^
+
+    Header 5
+    """"""""
+
+Horizontal Rules
+----------------
+
+Simply place four or more hyphens (``-``) on there own line with blank lines before and after, with no indent
+
+For example:
+
+``----``
+
+----
+
+Code Formatting
+---------------
+
+To use ``highlighted monospaced text``, prepend and append the text with a double backquote (``````).
+
+For example:
+
+``Change into directory with \``cd /etc/\`` and show a long list of of the directory with \``ls -l\``.``
```

Comment:
Hm, something broke in github with that line. I'll  look into it!

It was meant to be a code example with the mark up below it.

Reply:
Yeah, I realised that after writing the initial comment! I had a quick play and I couldn't get it to render properly on GH either. So it might actually just be a GH rendering issue?!

Feel free to have a look and see if you can fix it, but if not, I suggest that we just provide a note (in brackets?!) that it probably isn't displaying properly in GH?!

Reply:
Alternatively I can make it a code block :)

Reply:
> Alternatively I can make it a code block :)

Of course! I didn't even think of that. Yeah, that'd be better.